native/test/TemplateAppTest/src/com/salesforce/samples/templateapp/PasscodeActivityTest.java

```diff
@@ -443,9 +443,6 @@ private void doEditorAction(final int textViewId, final int actionCode) {
                     v.onEditorAction(actionCode);
                 }
             });
-            waitSome();
```

Comment:
recently, IME_ACTION_GO action can trigger key_down and key_up event successfully, so we don't need to send enter key separately as before, otherwise will trigger it twice and cause to enter empty passcode, which cause test failed.